WordPress/src/main/res/layout/plans_purchase_success_fragment.xml

```diff
@@ -36,7 +36,7 @@
             android:text="@string/dashboard_card_plans_checkout_success_title"
             android:textAlignment="viewStart"
             android:textFontWeight="700"
-            android:textSize="@dimen/m3_sys_typescale_headline_large_text_size"
+            android:textSize="@dimen/plans_purchase_success_title"
```

Comment:
👋 @AjeshRPai !

Question (❓): As per [this](https://github.com/wordpress-mobile/WordPress-Android/pull/18433/commits/032c245e08642a4b5f2c9472a555b14a851504f0) commit that I replaced this `m3_sys_typescale_headline_large_text_size` dimension with the more generic `text_sz_extra_extra_large` dimension instead of creating a new custom such `plans_purchase_success_title` dimension, why did you chose creating a new custom one instead of reusing the generic one?

Reply:
😄 @irfano just merged @ravishanker's #18539 PR, which actually fixes all that (see merge conflict)! 🤣

Reply:
I guess we can close this PR... 🤷 ✅

Reply:
Great, thanks for explaining the reasoning behind that decision @AjeshRPai , let me think about it a bit and I'll come back to you on that. 🙇  😊 👍

Reply:
> ...@AjeshRPai , let me think about it a bit and I'll come back to you on that.

TL;DR: I agree (`paqN3M-TC-p2#comment-1139`). 💯